dockstore-webservice/src/main/java/io/dockstore/webservice/languages/WDLHandler.java

```diff
@@ -163,7 +165,13 @@ public Version parseWorkflowContent(String filepath, String content, Set<SourceF
             } catch (WdlParser.SyntaxError ex) {
                 LOG.error("Unable to parse WDL file " + filepath, ex);
                 Map<String, String> validationMessageObject = new HashMap<>();
-                validationMessageObject.put(filepath, "WDL file is malformed or missing, cannot extract metadata. " + ex.getMessage());
+
+                if (wdlBridge.fileFirstLineStartsWithString(UNSUPPORTED_WDL_VERSION, tempMainDescriptor.getAbsolutePath())) {
```

Comment:
Think this should be flipped. What if the next version that someone experiments with ends up being 1.2 or 1.1.1?
Rather than checking that this is specifically 1.1 and treating that as a one off. This should use https://github.com/zafarkhaja/jsemver which I believe is in the classpath already to parse out the version number and report if the version is simply larger than the current WDL version.

Reply:
done